components/webext-storage/src/migration.rs

```diff
@@ -141,32 +155,52 @@ pub fn migrate(tx: &Transaction<'_>, filename: &Path) -> Result<usize> {
     // and the last one
     if last_ext_id != "" && !curr_values.is_empty() {
         // a different extension id - write what we have to the DB.
-        do_insert(tx, &last_ext_id, curr_values)?;
-        num_extensions += 1;
+        let entries = do_insert(tx, &last_ext_id, curr_values)?;
+        mi.extensions_successful += 1;
+        mi.entries_successful += entries;
     }
-    log::info!("migrated {} extensions", num_extensions);
-    Ok(num_extensions)
+    log::info!("migrated {} extensions: {:?}", mi.extensions_successful, mi);
```

Comment:
This is totally fine here, given that we'll want to move to `Log.jsm` logging eventually and it's just one very small struct, but there's a dev-platform post from a while bacl cautioning about printing with `#[derive(Debug)]`, since the extra code can add up, and the majority of our users won't see it.

Reply:
I think we'd want this in the sync-log tbh.